packages/experiments/src/components/Button/ButtonMenu/ButtonMenu.tsx

```diff
@@ -0,0 +1,23 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { withSlots, createComponent, getSlots } from '../../../Foundation';
+import { IButtonMenuComponent, IButtonMenuProps, IButtonMenuSlots } from './ButtonMenu.types';
+import { ButtonMenuStyles as styles } from './ButtonMenu.styles';
+
+const view: IButtonMenuComponent['view'] = props => {
+  const { menuType = 'div', ...rest } = props;
+
+  const Slots = getSlots<IButtonMenuProps, IButtonMenuSlots>(props, {
+    root: menuType
+  });
+
+  return <Slots.root {...rest} />;
+};
```

Comment:
Is this intended to be a legitimate viable default menu option? Are there any examples that use it? (It seem all the examples were converted to use ContextualMenu.)